r/inst/build_arrow_static.sh

```diff
@@ -82,5 +82,7 @@ ${CMAKE} -DARROW_BOOST_USE_SHARED=OFF \
     ${EXTRA_CMAKE_FLAGS} \
     -G ${CMAKE_GENERATOR:-"Unix Makefiles"} \
     ${SOURCE_DIR}
-${CMAKE} --build . --target install
+
+${CMAKE} --build . --parallel --target install
```

Comment:
Wait, so unless you add --parallel, it won't use the `-j` jobs set in MAKEFLAGS?

Reply:
Ok, it seems that wasn't necessary actually. MAKEFLAGS was simply not set correctly :-)